ArduPlane/quadplane.cpp

```diff
@@ -746,8 +749,7 @@ bool QuadPlane::setup(void)
     
 failed:
     initialised = false;
-    enable.set(0);
-    gcs().send_text(MAV_SEVERITY_INFO, "QuadPlane setup failed");
+    AP_BoardConfig::config_error("QuadPlane setup failed");
```

Comment:
If all errors call config_error with useful messages, this branch can be eliminated